PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -321,6 +325,20 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
 
         let settings = VPNSettings(defaults: .networkProtectionGroupDefaults)
 
+        Configuration.setURLProvider(VPNAgentConfigurationURLProvider())
+        configuratioManager = ConfigurationManager()
+        configuratioManager.start()
+        let privacyConfigurationManager = VPNPrivacyConfigurationManager.shared
+        // Load cached config (if any)
+        let configStore = ConfigurationStore()
+        privacyConfigurationManager.reload(etag: configStore.loadEtag(for: .privacyConfiguration), data: configStore.loadData(for: .privacyConfiguration))
+
+        if privacyConfigurationManager.privacyConfig.isSubfeatureEnabled(BackgroundAgentPixelTestSubfeature.pixelTest)
+            && !UserDefaults.configurationGroupDefaults.bool(forKey: BackgroundAgentPixelTestSubfeature.pixelTest.rawValue) {
+            Pixel.fire(pixel: .networkProtectionConfigurationPixelTest)
```

Comment:
Since this is in the init function of the VPN, it won't necessarily react to Privacy Config changes quickly - the VPN is intended to run indefinitely, so we'll want to instead observe config refreshes and fire it there.

We may be able to use the `UniquePixel` API as well, to save some UserDefaults boilerplate.

Reply:
I'd actually consider using the `DailyPixel` API here instead since it'll give us an accurate look each day at the exact rollout of the flag.